.github/workflows/ci-kubeadm.yml

```
@@ -96,7 +96,7 @@ jobs:
         run: kubectl proxy &
 
       - name: Run Auto Testing Framework
-        run: ./tests/test-scenarios-local.sh
+        run: ./tests/test-scenarios-local.sh -testAll -enableHostPolicy
```

```suggestion
        run: ./tests/test-scenarios-local.sh -testAll -enableHostPolicy
        timeout-minutes: 15
```
We should timeout the test run after 15 minutes to avoid it running infinitely.